packages/beeq/src/components/card/_storybook/bq-card.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import mdx from './bq-card.mdx';
+import { CARD_TYPE } from '../bq-card.types';
+
+const meta: Meta = {
+  title: 'Components/Card',
+  component: 'bq-card',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    type: { control: 'select', options: [...CARD_TYPE] },
+    content: { control: 'text', table: { disable: true } },
+  },
+  args: {
+    type: 'default',
+  },
+};
+export default meta;
+
+type Story = StoryObj;
+
+const Template = (args: Args) => html`
+  <div class="max-w-sm"><bq-card type=${ifDefined(args.type)}>${ifDefined(args.content)}</bq-card></div>
+`;
+
+export const Default: Story = {
+  render: Template,
+  args: {
+    content: html`
+      <div class="flex flex-col gap-6">
+        <div>
+          <div class="flex flex-row items-center gap-xs2">
+            <span class="text-text-secondary">Title</span> <bq-icon color="text--brand" size="16" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+          </div>
+          <h4>194</h4>
+        </div>
+        <div class="flex flex-col">
+          <p><span class="font-bold">120</span> days</p>
+          <p class="text-text-secondary">avg resolution</p>
+        </div>
+        <div class="flex flex-col">
+          <p><span class="font-bold">120</span> days</p>
+          <p class="text-text-secondary">avg resolution</p>
+        </div>
+      </div>
+    `,
+  },
+};
+
+export const Feature_Highlights: Story = {
```

Comment:
Please better use CamelCase in the Story names.
```suggestion
export const FeatureHighlights: Story = {
```
Same for the rest down here 🙂